### ISSUE TITLE
Improve Pre-Loader Animation

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/doc2pen#512]
[Reference](https://www.youtube.com/watch?v=C0oBg_J2gbY&feature=youtu.be)

- **DO NOT COPY PASTE DIRECTLY FROM THE REFERENCE**
- Use the name of our project
- Use our logo ( Favicon)
- Try to animate the pen in our logo
- Change the color of the background of the current animation to white, logo (with the animation) to sea green, text to black.
[Reference for color scheme](https://dribbble.com/shots/12425249-Wave)

**Comment by vedant-jain03** [comment]
I have gone throw your requirements and I think that I can fix this issue.
Please assign me so that I can work on this.

**Comment by smaranjitghose** [comment]
> I have gone throw your requirements and I think that I can fix this issue.
> Please assign me so that I can work on this.

Sure.
Does 4 days sound good for an initial PR?

**Comment by vedant-jain03** [comment]
Yes 4 days are enough for this  and also Thanks a lot for assigning me this issue.
I also have one query that should I make PR after 8 march or before also it will count a point in GSSOC.

**Comment by smaranjitghose** [comment]
> Yes 4 days are enough for this and also Thanks a lot for assigning me this issue.
> I also have one query that should I make PR after 8 march or before also it will count a point in GSSOC.

Sure, then start the work from now. Give the PR after 8th. This way you would have a headstart

**Comment by vedant-jain03** [comment]
Okhay I have started working on this. 
Once again thanks a lot for assigning me✌

**Comment by vedant-jain03** [comment]
Hey, @smaranjitghose I have started working on this issue and I just need to know the directory or png file of your favicon

**Comment by vedant-jain03** [comment]
![bandicam 2021-03-08 11-31-36-163](https://user-images.githubusercontent.com/76901313/110281031-e3cb3480-8001-11eb-8f85-dcc65d6cf30e.jpg)

@smaranjitghose  I Have done the animation part of it. I want to know that if this combination is good?
I have involved:
->Logo(pen)
->Doc2pen(name of org.)
as you have mentioned in the issue
I have changed the color of the pen into sea green as well.
If it sounds ok So should I make PR or any changes I should make more on it.

**Comment by smaranjitghose** [comment]
> ![bandicam 2021-03-08 11-31-36-163](https://user-images.githubusercontent.com/76901313/110281031-e3cb3480-8001-11eb-8f85-dcc65d6cf30e.jpg)
> 
> @smaranjitghose I Have done the animation part of it. I want to know that if this combination is good?
> I have involved:
> ->Logo(pen)
> ->Doc2pen(name of org.)
> as you have mentioned in the issue
> I have changed the color of the pen into sea green as well.
> If it sounds ok So should I make PR or any changes I should make more on it.

Will be addressed in the PR😊
